backend/service/aws/dynamodb.go

```diff
@@ -140,6 +141,15 @@ func newProtoForTable(t *types.TableDescription, account, region string) *dynamo
 	return ret
 }
 
+func newProtoForContinuousBackups(t *types.ContinuousBackupsDescription, account string, region string) *dynamodbv1.ContinuousBackups {
```

Comment:
from the docs, looks like `ContinuousBackupsStatus` is required to be in the description output. is that true for `PointInTimeRecoveryDescription`, do we need check for nil?

Reply:
Good catch tyty, I'll add nil checks for `PointInTimeRecoveryDescription` and its fields